### ISSUE TITLE
Pipes won't place at certain height

### ISSUE THREAD

**Issue by Ginge2001** [BuildCraft/BuildCraft#4222]
At y co-ordinate 128, no buildcraft pipes will place. All layers below and seemingly all layers above work fine.

BuildCraft version: 2.2.14
Forge version: 3.3.8.164
Link to crash report or log: 
Singleplayer or multiplayer: singleplayer
Steps to reproduce: Attempt to place pipes at y co-ordinate 128.
Additional information:

**Comment by AlexIIL** [comment]
Hang on, Buildcraft 2.2.14? What version of minecraft is that running on?

**Comment by AlexIIL** [comment]
Either way I'd assume it's because of this check here not being quite right:

https://github.com/BuildCraft/BuildCraft/blob/90a7ea3baa630948a1c52b4788d2652d0f78ee96/common/net/minecraft/src/buildcraft/transport/ItemPipe.java#L68

**Comment by Ginge2001** [comment]
My version is 1.25. Any solutions/workarounds for this issue?

**Comment by AlexIIL** [comment]
I'm not sure really, I wasn't around then and I'm not sure who to ping for help. Fortunately it's only a single y level, so would it be possible to place chests there instead and just use wooden pipes for moving items out?

**Comment by Generalcamo** [comment]
It was probably never properly accounted for the build height limit change in 1.2. Before that it was half, and this code would be correct.

**Comment by Ginge2001** [comment]
Yeah I'm just using chests for the time being. I didn't think it'd be an issue since the Yogscast have reliably built Tekkit structures that are high up.
